examples/vllm-deepseek-coder-1-3b-instruct/README.md

```diff
@@ -0,0 +1,147 @@
+# Deploy LLM models with vLLM
```

Comment:
In order to follow the same organisation and style in the repository, I suggest you move this example to folder expose_services and rename this folder to something simpler like "vllm-deepseek"